src/cli/commands/upgrade-interactive.js

```diff
@@ -4,47 +4,52 @@ import type {Dependency} from '../../types.js';
 import type {Reporter} from '../../reporters/index.js';
 import type Config from '../../config.js';
 import inquirer from 'inquirer';
-import PackageRequest from '../../package-request.js';
-import {Add} from './add.js';
-import {Install} from './install.js';
 import Lockfile from '../../lockfile/wrapper.js';
+import {Add} from './add.js';
+import {getOutdated, setFlags as setUpgradeFlags} from './upgrade.js';
 
 export const requireLockfile = true;
 
 export function setFlags(commander: Object) {
-  commander.usage('upgrade-interactive');
-  commander.option('-E, --exact', 'upgrade to most recent release with exact version');
-  commander.option('-T, --tilde', 'upgrade to most recent release with patch version');
+  setUpgradeFlags(commander);
```

Comment:
Yeah I was just trying to avoid the duplication. I can put the flags in both commands though.